File: client/src/app/_models/UserParams.ts
import { User } from './user';

export class UserParams {
  appUserType: string;
  major = 'Accounting';
  position = 'Finance';
  pageNumber = 1;
  pageSize = 50;
  orderBy = 'lastActive';

  constructor(user: User) {
    this.appUserType =
      user.appUserType === 'ColStudent' ? 'EmpHr' : 'ColStudent';
  }
}

File: API/Helpers/UserParams.cs
namespace API.Helpers
{
    public class UserParams : PaginationParams
    {

        public string CurrentUsername { get; set; }
        public string AppUserType { get; set; }

        public string Major { get; set; } = "Accounting";

        public string Position { get; set; } = "Finance";

        public string OrderBy { get; set; } = "created";

    }
}